x/auth/client/cli/tx_multisign.go

```diff
@@ -127,7 +136,7 @@ func makeMultiSignCmd() func(cmd *cobra.Command, args []string) (err error) {
 				}
 			}
 		}
-
+		// TODO how to deal with multisigPub and multisigSig ?
```

Comment:
Just wonder if `AddSignatureV2` at line 134 makes sense or has to be deleted?

Reply:
It still makes sense, the multisign logic should not be changed.

Reply:
Keyring is only about getting and saving keys. Keyring users (all packages using keyring) should not know if Keyring is using protobuf under the hood. Externally, Keyring will return data structures, not proto/amino bytes.
 
So, Keyring migration should not impact at all any other use-case.